neonvm/runner/main.go

```diff
@@ -620,7 +624,10 @@ func newConfig(logger *zap.Logger) *Config {
 		cfg.appendKernelCmdline, "Additional kernel command line arguments")
 	flag.BoolVar(&cfg.skipCgroupManagement, "skip-cgroup-management",
 		cfg.skipCgroupManagement,
-		"Don't try to manage CPU (use if running alongside container-mgr)")
+		"Don't try to manage CPU (use if running alongside container-mgr, or if dummy CPU server is enabled)")
+	flag.BoolVar(&cfg.enableDummyCPUServer, "enable-dummy-cpu-server",
```

Comment:
Do we actually plan to use runner without the enable-dummy-cpu-server?

I don't mind that it is there, but I think I'd run the server unconditionally, until we (likely) decide to abandon it overall.

Reply:
> Do we actually plan to use runner without the enable-dummy-cpu-server?

For now, yes. We haven't come to a decision beyond that.